tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyAjax.java

```diff
@@ -547,6 +547,7 @@ public CheckCLDR handleCheck(String path, String fullPath, String value, Options
                         r.put(SurveyMain.QUERY_VALUE_SUFFIX, mySession.settings().get(pref, null));
                         send(r, out);
                     } else if (what.equals(WHAT_VSUMMARY)) {
+                        // TODO: obviated by org.unicode.cldr.web.api.PriorityItems
```

Comment:
the 'old' JS could probably call the new API and we delete this.  I'm inclined to leave it alone until we remove dojo.